### ISSUE TITLE
A question about model selection

### ISSUE THREAD

**Issue by peter943** [uchidalab/dtw-features-ucr#1]
Hi,
I have learnt your paper these day, which is very enlightening to me. I have a question about model selection. In your paper, you say, most of the UCR datasets have very small training sets, which is generally not ideal for CNN and deep learning solutions. 

I do not know if I want to evaluate my method in UCR, how should I choose the model. Maybe the model corresponding to the minimum training loss or maximum test accuracy. Could you please give me some help? Thanks a lot.
